recipes/quarticRootsFlocke/meta.yaml

```diff
@@ -26,6 +26,7 @@ source:
     - patches/0015-Generate-Config-files.patch
     - patches/0016-Use-standard-generated-library-filename.patch
     - patches/0017-Fix-package-version-when-not-in-a-git-repo.patch
+    - patches/0018-Fix-QUARTIC_INCLUDE_DIRS-to-not-include-lib.patch
```

Comment:
Do you think that these changes or cusotmizations could be suggested in a PR to be included in the upstream repo?

Reply:
Yes, many of them could indeed. I was just worried about the time it would take for them to be reviewed merged as I needed this package ON ASAP.